public/app/features/panel/panel_menu.js

```diff
@@ -14,6 +14,7 @@ function (angular, $, _, Tether) {
           '<span class="panel-title drag-handle pointer">' +
             '<span class="icon-gf panel-alert-icon"></span>' +
             '<span class="panel-title-text drag-handle">{{ctrl.panel.title | interpolateTemplateVars:this}}</span>' +
+            '<span class="panel-help-text"><info-popover mode="bold">{{ctrl.panel.helpText}}</info-popover></span>' +
```

Comment:
I am a bit worried about adding it like this, since it will slow down dashboard loading. Maybe some other way to add it after,  and only when helpText is present, this way info-popover will always have to be compiled and added. What happens if there is no helpText? want to keep the panel bindings/rendering as light weight as possible. But this solution is simple / easy to maintain so might still be best